test/libsolidity/smtCheckerTests/types/address_transfer_3.sol

```diff
@@ -0,0 +1,15 @@
+contract C
+{
+	function f(address payable a) public {
+		require(1000 == address(this).balance);
+		require(100 == a.balance);
+		a.transfer(600);
```

Comment:
But `transfer` can forward the eth to another contract, can't it?

Reply:
Not sure about this. There is the 2300 gas stipend, and `call` is `2600` cold. But it can be made `100` by using access lists. Let me check if all value forwarding calls needs to use at least `2300`.

Reply:
Not sure what exactly you mean. The goal of `a.transfer(600)` is exactly to send 600 wei to `a`

Reply:
I meant that the condition `assert(a.balance == 700);` doesn't have to be true.

Reply:
Ohh that's true

Reply:
But I'm not able to test it properly: https://gist.github.com/hrkrshnn/0066038f2e740a999290a1861d1118b5

Reply:
and that's not working?

Reply:
We concluded [here](https://matrix.to/#/!poXqlbVpQfXKWGseLY:gitter.im/$812or6L6BR2cUPPCQ9aPIqOknLfm7MMo6l8ImZR6s20?via=gitter.im&via=matrix.org&via=ekpyron.org) that this is not a problem :)